docker/helm/templates/kyuubi-deployment.yaml

```diff
@@ -46,6 +46,8 @@ spec:
             - name: frontend-port
               containerPort: {{ .Values.server.bind.port }}
               protocol: TCP
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
```

Comment:
shall we use `indent` rather than `nindent` here ? otherwise we will get a new empty line. I just read some [related docs](https://helm.sh/docs/chart_template_guide/function_list/#nindent).

Reply:
> shall we use `indent` rather than `nindent` here ? otherwise we will get a new empty line. I just read some [related docs](https://helm.sh/docs/chart_template_guide/function_list/#nindent).

It seems that it must be `nindent` here, if use `indent`, we will get error ` mapping values are not allowed in this context`.